<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1208</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Sponsors</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabSponsor">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>30</y>
      <width>1231</width>
      <height>611</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Fournisseur</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>1161</width>
        <height>571</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 238);
background-color: rgb(233, 255, 254);</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QTableView" name="tab_employes">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>240</y>
         <width>501</width>
         <height>261</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>461</width>
         <height>491</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="label_ID">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>OCR A Extended</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Nom employes:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_Nom">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>161</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>OCR A Extended</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>prenom employes:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_Adr">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>151</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>OCR A Extended</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ID :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_nomE">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>39</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(35, 40, 3, 255), stop:0.16 rgba(136, 106, 22, 255), stop:0.225 rgba(166, 140, 41, 255), stop:0.285 rgba(204, 181, 74, 255), stop:0.345 rgba(235, 219, 102, 255), stop:0.415 rgba(245, 236, 112, 255), stop:0.52 rgba(209, 190, 76, 255), stop:0.57 rgba(187, 156, 51, 255), stop:0.635 rgba(168, 142, 42, 255), stop:0.695 rgba(202, 174, 68, 255), stop:0.75 rgba(218, 202, 86, 255), stop:0.815 rgba(208, 187, 73, 255), stop:0.88 rgba(187, 156, 51, 255), stop:0.935 rgba(137, 108, 26, 255), stop:1 rgba(35, 40, 3, 255));
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_PrenomE">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>79</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 207);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_ajouter">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>400</y>
          <width>161</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>OCR A Extended</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">
color: rgb(35, 170, 102);
</string>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_Adr_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>151</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>OCR A Extended</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Mail :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_mail">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>159</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_modifier">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>400</y>
          <width>161</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>OCR A Extended</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">
color: rgb(35, 170, 102);
</string>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>210</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Séléctionnez une image</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_img_2">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>250</y>
          <width>161</width>
          <height>161</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">capacity:10%;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_salaire">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>200</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_Adr_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>200</y>
          <width>151</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>OCR A Extended</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Salaire :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_Adr_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>290</y>
          <width>151</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>OCR A Extended</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Heure de travail : </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_Adr_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>250</y>
          <width>151</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>OCR A Extended</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Poste :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_poste">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>250</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_HDT">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>290</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_ID">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>120</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 207);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
       </widget>
       <widget class="QPushButton" name="Ard_button_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>131</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>OCR A Extended</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(216, 191, 0);
color: rgb(35, 170, 102);</string>
        </property>
        <property name="text">
         <string>TEST</string>
        </property>
       </widget>
      </widget>
      <widget class="QLineEdit" name="lineEdit_id_sup">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>520</y>
         <width>181</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_supp">
       <property name="geometry">
        <rect>
         <x>720</x>
         <y>510</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>OCR A Extended</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(35, 170, 102);</string>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>520</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ID :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_recherche">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>220</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ID :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_recherche">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>220</y>
         <width>181</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(35, 40, 3, 255), stop:0.16 rgba(136, 106, 22, 255), stop:0.225 rgba(166, 140, 41, 255), stop:0.285 rgba(204, 181, 74, 255), stop:0.345 rgba(235, 219, 102, 255), stop:0.415 rgba(245, 236, 112, 255), stop:0.52 rgba(209, 190, 76, 255), stop:0.57 rgba(187, 156, 51, 255), stop:0.635 rgba(168, 142, 42, 255), stop:0.695 rgba(202, 174, 68, 255), stop:0.75 rgba(218, 202, 86, 255), stop:0.815 rgba(208, 187, 73, 255), stop:0.88 rgba(187, 156, 51, 255), stop:0.935 rgba(137, 108, 26, 255), stop:1 rgba(35, 40, 3, 255));
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_recherche">
       <property name="geometry">
        <rect>
         <x>850</x>
         <y>200</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>OCR A Extended</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(216, 191, 0);
color: rgb(35, 170, 102);</string>
       </property>
       <property name="text">
        <string>Recherche</string>
       </property>
      </widget>
      <widget class="QLabel" name="countdown">
       <property name="geometry">
        <rect>
         <x>980</x>
         <y>10</y>
         <width>131</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_trier">
       <property name="geometry">
        <rect>
         <x>980</x>
         <y>360</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>OCR A Extended</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: rgb(35, 170, 102);
</string>
       </property>
       <property name="text">
        <string>TRIER</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_pdf">
       <property name="geometry">
        <rect>
         <x>1010</x>
         <y>180</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>OCR A Extended</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(216, 191, 0);
color: rgb(35, 170, 102);</string>
       </property>
       <property name="text">
        <string>PDF</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_showStatsButton">
       <property name="geometry">
        <rect>
         <x>930</x>
         <y>510</y>
         <width>201</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>OCR A Extended</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: rgb(35, 170, 102);
</string>
       </property>
       <property name="text">
        <string>statistique</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_qr">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>90</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>QR</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Ard_button">
       <property name="geometry">
        <rect>
         <x>1000</x>
         <y>270</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>OCR A Extended</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(216, 191, 0);
color: rgb(35, 170, 102);</string>
       </property>
       <property name="text">
        <string>TEST</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>160</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1208</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSponsor">
    <property name="title">
     <string>foursn</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionNew"/>
    <addaction name="actionGen_rer_PDF"/>
    <addaction name="actionStatistiques"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuSponsor"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normaloff>../../design/new icon.png</normaloff>../../design/new icon.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>../../design/close-icon.png</normaloff>../../design/close-icon.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionGen_rer_PDF">
   <property name="icon">
    <iconset>
     <normaloff>../../design/pdf_icon.png</normaloff>../../design/pdf_icon.png</iconset>
   </property>
   <property name="text">
    <string>Genérer PDF</string>
   </property>
  </action>
  <action name="actionStatistiques">
   <property name="icon">
    <iconset>
     <normaloff>../../design/stats.png</normaloff>../../design/stats.png</iconset>
   </property>
   <property name="text">
    <string>Statistiques</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
